Makefile

```diff
@@ -136,10 +138,13 @@ tarballs-release: $(PROMU)
 	$(PROMU) checksum .tarballs
 	$(PROMU) release .tarballs
 
+.PHONY: package-hdfs
+package-hdfs: .hdfs.$(GOHDFS_VERSION).tar.gz
```

Comment:
This is part of the vendor hack. I use it to package the HDFS client using go 1.11 in a separate CI job, so that it can be used during the go 1.10 build. It's just there so that the CI job can solely execute the packaging, without anything else, and without having to know the exact git revision. So this is nothing that will stay around, once go modules are available.